modular_skyrat/master_files/code/modules/clothing/under/akula_jobs.dm

```diff
@@ -0,0 +1,302 @@
+/// How many wetstacks does the clothing's status effect apply to its wearer
+#define STATUS_EFFECT_STACKS 15
+
+/// A proc to be used in `equipped()` for all akula clothing which has the 'special tech' to keep their wearers slippery
+/obj/item/clothing/proc/apply_wetsuit_status_effect(mob/living/carbon/human/user)
```

Comment:
Also, this should be made into a component, instead of being a proc on the `/obj/item/clothing` level.

Reply:
Applying a component to a suit, which then applies a status_effect to its wearer (wearer isn't a variable btw you have to use loc) wasn't sane, nor functional. It just broke everything without giving proper debug feedback.